### ISSUE TITLE
Question about the training and testing details

### ISSUE THREAD

**Issue by heshuting555** [lightas/ICCV19_Pose_Guided_Occluded_Person_ReID#9]
Dear Jiaxu,

When reading your paper "Pose-Guided Feature Alignment for Occluded Person Re-Identification", I have some questions about the training and testing details about Occluded-DukeMTMC.

Do you train on the Occluded-DukeMTMC Dataset and then test on it? 

Thank you for reading and look forward to your reply!

**Comment by lightas** [comment]
@heshuting555 Yes, I trained on the training set of the Occluded-DukeMTMC Dataset and then tested on the test set.

**Comment by heshuting555** [comment]
OK,
Thank you for your reply.
